app/js/views/main.js

```diff
@@ -104,6 +104,29 @@ var mainViewTemplate =
 
     --checkbox-border-color: var(--background-minus-minus);
   }
+
+  fxos-customizer {
```

Comment:
Yes, I think all of this should go in the element's ```<style scoped>```. I don't think it's unfair to make some assumptions about the use cases for this. The ```::host``` selector can be overridden by client content if they don't like our decisions. Using CSS vars makes sense, though.